src/components/UI/Buttons/Buttons.tsx

```diff
@@ -1,19 +1,66 @@
 import { JSX } from 'preact';
 import styles from './Buttons.module.sass';
+import backIcon from './icons/backArrow.svg';
+import nextIcon from './icons/nextArrow.svg';
 
 interface ButtonProps extends JSX.HTMLAttributes<HTMLButtonElement> {
   onClick: () => void;
   children: JSX.Element | string;
 }
 
 /**
- * Button components for the UI
+ * Basic Button component
  */
+export function Button(props: ButtonProps) {
+  return <button className={styles.button}>{props.children}</button>;
```

Comment:
Add `props` to the button element.

```suggestion
  return <button className={styles.button} {...props}>{props.children}</button>;
```